setup.py

```diff
@@ -69,19 +69,19 @@
     ],
     'test': [
         'pytest',
-        'mock',  # it would be better for 'test:python_version in 2.7'
-        'simplejson',  # better: 'test:platform_python_implementation=="PyPy"'
+        'pytest-cov',
         'html5lib',
     ],
+    'test:python_version<"3"': [
+        'enum34',
+        'mock',
+    ],
+    'test:python_version>="3"': [
+        'mypy',
+        'typed_ast',
+    ],
 }
 
-# for sdist installation with pip-1.5.6
-if sys.platform == 'win32':
-    requires.append('colorama>=0.3.5')
-
-if sys.version_info < (3, 5):
-    requires.append('typing')
```

Comment:
I think this removing lines means 'drop support pip-1.5.6'.
Maybe such environment is very rare nowadays, however it would be better if it is indicated on CHANGES.

Reply:
@shimizukawa I've refrained from touching CHANGES because it causes merge conflicts and you guys usually do it. Should I do this or can I simply call it out in the commit message as I've done and leave the CHANGES update to maintainers?

On a side note, have you seen [`reno`](https://docs.openstack.org/reno/latest/). Is this something you might be open to using in Sphinx in a future release?

Reply:
Okay, I'll update CHANGES on merging.